commit b1b4f7209ecaad4bf9a5d0d2ef1338409d364bac
Author: Joseph Myers <joseph@codesourcery.com>
Date:   Thu Jul 1 17:37:36 2021 +0000

    Update syscall lists for Linux 5.13
    
    Linux 5.13 has three new syscalls (landlock_create_ruleset,
    landlock_add_rule, landlock_restrict_self).  Update syscall-names.list
    and regenerate the arch-syscall.h headers with build-many-glibcs.py
    update-syscalls.
    
    Tested with build-many-glibcs.py.

Modified to only update syscall-names.list to Linux 5.13.

diff --git a/sysdeps/unix/sysv/linux/syscall-names.list b/sysdeps/unix/sysv/linux/syscall-names.list
index 8e3cfa0e77..89c5895b9b 100644
--- a/sysdeps/unix/sysv/linux/syscall-names.list
+++ b/sysdeps/unix/sysv/linux/syscall-names.list
@@ -21,8 +21,8 @@
 # This file can list all potential system calls.  The names are only
 # used if the installed kernel headers also provide them.
 
-# The list of system calls is current as of Linux 5.12.
-kernel 5.12
+# The list of system calls is current as of Linux 5.13.
+kernel 5.13
 
 FAST_atomic_update
 FAST_cmpxchg
@@ -224,6 +224,9 @@ kexec_file_load
 kexec_load
 keyctl
 kill
+landlock_add_rule
+landlock_create_ruleset
+landlock_restrict_self
 lchown
 lchown32
 lgetxattr
